### DISCUSSION TITLE
Mobile hacks (sort of!) (UNOFFICIAL PROJECT)

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#487] [(A) Anything]
Hi, I've created a station for you to request a hacked account if you use a mobile device. I do not hack existing accounts, because, in the event that my email was compromised, any account logins sent to me would be compromised as well, so I do not want to risk your security- I am sorry for any incovience this causes.

Use [this form](https://nameisa.github.io/MobileDevice-ProdigyHacking/) to request a hacked account. You can have one account for free, but I really don't want to go out of my way to make excessive alts for people for no reason, so if you want a bunch of accounts, that costs a fee. However, if your account is broken, I will attempt to repair it for free. If I am unable to repair it, you will get a new account for free.

This is not an official service of PMGH, it's made by myself - a community member - as PMGH does not support giving out/logging into accounts for others [for a variety of legitimate security reasons](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/discussions/170), which is why I suggest you to consider your choice before requesting me to create an account. Specifically,
* If my email was compromised, so are all the accounts shared in it.
* It is up to you to trust if you want to trust me with the login details to your new hacked account.

Note that orders may take a bit, I have been quite busy recently.

~NameIsA

(this project has a repository, but it's just a README and some HTML code. https://github.com/NameIsA/MobileDevice-ProdigyHacking/tree/github-pages)

**Comment by irhrhd** [comment]
When are you gonna start this again

**Reply by ghost** [reply]
Haven't worked on it recently, but other users have been helping with it on the repository. CRobbins0867 (I think that's his username?) has been supplying accounts, so feel free to ask him.

**Reply by SakuraKitsunee** [reply]
@NameIsA yessir that's my name!

**Comment by afkvido** [comment]
Did you delete it?

**Reply by afkvido** [reply]
the links don't work
